rollup/circuitcapacitychecker/libzkp/src/lib.rs

```diff
@@ -57,39 +58,7 @@ pub mod checker {
     /// # Safety
     #[no_mangle]
     pub unsafe extern "C" fn apply_tx(id: u64, tx_traces: *const c_char) -> *const c_char {
-        let result = panic::catch_unwind(|| {
```

Comment:
As I said, we should not remove `panic::catch_unwind` in case there's an unexpected panic inside the ccc